Remove "custom rule that disregards any fancy flags that the
user might have set" and ../quicktime/libquicktimehv.la

--- a/guicast/Makefile.am
+++ b/guicast/Makefile.am
@@ -1,8 +1,8 @@
 lib_LTLIBRARIES = libguicast.la
 noinst_PROGRAMS = bootstrap pngtoh
 
-libguicast_la_LIBADD = $(OPENGL_LIBS) -lXxf86vm -lXv -lXext -lX11 -lpng $(X_EXTRA_LIBS) $(XFT_LIBS)
-libguicast_la_LDFLAGS = $(X_LIBS) -version-info 1:0:0 
+libguicast_la_LIBADD = $(OPENGL_LIBS) -lXxf86vm -lXv -lXext -lX11 -lpng ../quicktime/libquicktimehv.la $(X_EXTRA_LIBS) $(XFT_LIBS)
+libguicast_la_LDFLAGS = $(X_LIBS) -version-info 1:0:0
 libguicast_la_SOURCES = \
 	bcbar.C \
 	bcbitmap.C \
@@ -210,18 +210,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/quicktime
 AM_CXXFLAGS = $(LARGEFILE_CFLAGS) $(X_CFLAGS)
 LIBTOOL = $(SHELL) $(top_builddir)/libtool $(LTCXX_FLAGS)
 
-# special CFLAGS that can be set on the make command line
-BOOTSTRAP_CFLAGS=
 bootstrap_SOURCES=bootstrap.c
-bootstrap_LDADD =
-
-# custom rule that disregards any fancy flags that the user might have set
-bootstrap.$(OBJEXT): bootstrap.c
-	$(CC) $(BOOTSTRAP_CFLAGS) -c $<
 
 pngtoh_SOURCES = pngtoh.c
 
-pngtoh$(EXEEXT): pngtoh.c
-	$(CC) -o pngtoh$(EXEEXT) $<
-
 EXTRA_DIST = images
